Source/src/core/rendering/Skybox.cpp

```diff
@@ -60,6 +60,87 @@ void Hachiko::Skybox::ReleaseCubemap()
     cube.loaded = false;
 }
 
+void Hachiko::Skybox::DrawImGui() 
+{
+    ImGui::PushID(this);
+
+    ImGui::Text("Skybox");
+
+    ImGui::TextDisabled("Keep in mind that all images \nneed to have the same format!");
+
+    if (ImGui::CollapsingHeader("Params"))
+    {
+        SelectSkyboxTexture(TextureCube::Side::RIGHT);
+        SelectSkyboxTexture(TextureCube::Side::LEFT);
+        SelectSkyboxTexture(TextureCube::Side::BOTTOM);
+        SelectSkyboxTexture(TextureCube::Side::TOP);
+        SelectSkyboxTexture(TextureCube::Side::CENTER);
+        SelectSkyboxTexture(TextureCube::Side::BACK);
+    }
+
+    ImGui::PopID();
+}
+
+const std::string TextureCubeSideToString(Hachiko::TextureCube::Side cube_side)
+{
+    switch (cube_side)
+    {
+    case Hachiko::TextureCube::Side::RIGHT:
+        return "Right";
+    case Hachiko::TextureCube::Side::LEFT:
+        return "Left";
+    case Hachiko::TextureCube::Side::BOTTOM:
+        return "Bottom";
+    case Hachiko::TextureCube::Side::TOP:
+        return "Top";
+    case Hachiko::TextureCube::Side::CENTER:
+        return "Center";
+    case Hachiko::TextureCube::Side::BACK:
+        return "Back";
+    }
+}
+
+void Hachiko::Skybox::SelectSkyboxTexture(TextureCube::Side cube_side)
+{
+    const std::string title = Hachiko::StringUtils::Concat("Select texture ", TextureCubeSideToString(cube_side));
+    const char* filters = "Image files{.png,.tif,.jpg,.tga}";
+
+    if (ImGui::Button(Hachiko::StringUtils::Concat(TextureCubeSideToString(cube_side).c_str(), " Texture").c_str()))
+    {
+        ImGuiFileDialog::Instance()->OpenDialog(title.c_str(),
```

Comment:
Can OpenDialog call indentation be cleaner? Maybe two or three lines is enough and cleaner to see. I know in other places is the same (and I think it was me tbh) but we can fix them too.